### PR TITLE
Chapter 9 cleanup and removed doc reference to disablejsr for #706.

### PR DESCRIPTION
**Pull request by WalkerWatch**
Signed-off-by: WalkerWatch <ctwalker@gmail.com>

### CODE REVIEW

File: jetty-documentation/src/main/asciidoc/administration/startup/startup-unix-service.adoc [hunk 1]
```diff
@@ -72,30 +65,24 @@ Prepare some empty directories to work with.
...
-  running Jetty server
-
-Jetty $\{project.version} requires Java 7 (or greater) to run. Make sure
-you have it installed.
+This is the temporary directory assigned to Java by the Service Layer (this is what Java sees as the `java.io.tmpdir` System Property).
++
+This is intentionally kept separate from the standard temp directory of `/tmp`, as this location doubles as the Servlet Spec work directory.
+It is our experience that the standard temp directory is often managed by various cleanup scripts that wreak havoc on a long running Jetty server.
+
+Jetty $\{project.version} requires Java 7 (or greater) to run.
```
Comment (janbartel) [inline jetty-documentation/src/main/asciidoc/administration/startup/startup-unix-service.adoc]: The $\{project.version} does not substitute for anything on the generated doc.

Also, jetty-9.3 and above requires Java 8 or above to run.

File: jetty-documentation/src/main/asciidoc/administration/startup/startup-windows-service.adoc [hunk 2]
```diff
@@ -17,15 +17,11 @@
...
+There are no components that ship with the Jetty Distribution to make it a formal Windows Service.
 
-However, we recommend the use of
-https://commons.apache.org/proper/commons-daemon/procrun.html[Apache
-ProcRun's Daemon].
+However, we recommend the use of https://commons.apache.org/proper/commons-daemon/procrun.html[Apache ProcRun's Daemon].
 
-The techniques outlined here are based on Windows 7 (64-bit), using JDK
-7 (64-bit), running on an Intel i7 architecture machine.
+The techniques outlined here are based on Windows 7 (64-bit), using JDK 7 (64-bit), running on an Intel i7 architecture machine.
```
Comment (janbartel) [inline jetty-documentation/src/main/asciidoc/administration/startup/startup-windows-service.adoc]: jetty 9.3 and above needs JDK 8.

File: jetty-documentation/src/main/asciidoc/administration/startup/startup-windows-service.adoc [hunk 3]
```diff
@@ -55,28 +51,21 @@ C:\opt> dir
...
-  This is intentionally kept separate from the standard temp directories
-  of Windows, as this location doubles as the Servlet Spec work
-  directory.
+This is the temporary directory assigned to Java by the Service Layer (this is what Java sees as the `java.io.tmpdir` System Property).
++
+This is intentionally kept separate from the standard temp directories of Windows, as this location doubles as the Servlet Spec work directory.
 
-Or download Java 7 from:
-http://www.oracle.com/technetwork/java/javase/downloads/index.html
+Or download Java 7 from: http://www.oracle.com/technetwork/java/javase/downloads/index.html
```
Comment (janbartel) [inline jetty-documentation/src/main/asciidoc/administration/startup/startup-windows-service.adoc]: Java 8.

File: jetty-documentation/src/main/asciidoc/configuring/jsp/configuring-jsp.adoc [hunk 4]
```diff
@@ -179,13 +179,6 @@ By default, the Glassfish JSP container tries to use the compiler that is part o
 If you do not have a full JDK, you can configure the Eclipse Java Compiler that Jetty ships in the distro in `{$jetty.home}/lib/jsp/.`
 You need to define a SystemProperty that prevents the Glassfish JSP engine from defaulting to the in-JVM compiler.
```
Comment (janbartel) [inline jetty-documentation/src/main/asciidoc/configuring/jsp/configuring-jsp.adoc]: We should remove all discussion of Glassfish JSP going forward, as that is only valid for older versions of jetty.

**Comment by WalkerWatch** [comment]
@janbartel Included fixes to your comments in 49f55cf5fccb7389c0087c73f33a2b0f553c4203

**Pull request merged**